Misc/NEWS.d/next/Library/2017-07-24-23-52-39.bpo-30931.mZMsJ7.rst

```diff
@@ -0,0 +1,3 @@
+asyncore: poll() and poll2() functions now check if a dispatcher was closed (if
+it's file descriptor is no more in the map) before calling its handlers:
+handlers of closed dispatchers are no more called.
```

Comment:
@nirs: Oops, you are right. I rewrote the NEWS entry.

@csabella: fixed, thanks.